### PR TITLE
monsters2cats

### PR DESCRIPTION
**Pull request by chrissyrissmeyer**
Changes Dracula to Catsy and Wolfman to Luke for episodes 1-3. Also updates Dracula/Catsy pronouns.

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -3,22 +3,20 @@ layout: lesson
...
 
-Wolfman and Dracula have been hired by Universal Missions (a space
-services spinoff from Euphoric State University) to investigate if it
-is possible to send their next planetary lander to Mars.  They want to
-be able to work on the plans at the same time, but they have run into
-problems doing this in the past.  If they take turns, each one will
-spend a lot of time waiting for the other to finish, but if they work
-on their own copies and email changes back and forth things will be
-lost, overwritten, or duplicated.
+Catsy Cline and Luke Skywhisker have been asked to reconcile and normalize their digital
```
Comment (dunn) [inline index.md:6]: What's your editor's default line length?

**Review by mcritchlow — COMMENTED** [review]
Looks great to me, other than the hard line breaks that your editor seems to be introducing as @dunn noted.

**Comment by dunn** [comment]
We want hard breaks, just at the right point.

**Comment by mcritchlow** [comment]
Mine are at 120 for MD right now. I can change to 80, or whatever we want to settle on.

**Review by dunn — APPROVED** [review]

**Pull request merged**